components/organisms/ContributorCard/contributor-card.tsx

```diff
@@ -58,7 +58,7 @@ const ContributorCard = ({ className, contributor, topic, repositories, range }:
           </div>
         </div>
         <div className="h-32">
-          <CardLineChart lineChartOption={chart} />
+          <CardLineChart lineChartOption={chart} className="!h-36" />
```

Comment:
refactor (blocking): There's no need to use `!important`. Set a max height instead.

```suggestion
          <CardLineChart lineChartOption={chart} className="max-h-36" />
```